optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/shift/ShiftRestServiceImpl.java

```diff
@@ -205,7 +205,7 @@ public ShiftTemplate getTemplate(Integer tenantId) {
             return null;
         } else if (1 != result.size()) {
             throw new IllegalStateException("Each tenant can only have 1 template! Found " + result.size()
-                    + "templates!");
+                                                    + "templates!");
```

Comment:
Can someone please explain me what's that aligning with? :)

Reply:
It's two indents after the start of the String.. now why is that so, I really can't tell :)